### ISSUE TITLE
Supervision model MVP

### ISSUE THREAD

**Issue by LauraFredrich** [MeMo-KBC/bio-medRxiv#17]
Fonduer Tutorials: https://github.com/HazyResearch/fonduer-tutorials

**Comment by thuettel** [comment]
@macmacpri and I spend quite a while in understanding the necessary steps with deploying machine learning models based on (gold)labels and the labeling functions. We started drafting some .py scripts, but they're not ready to run yet. 

Some questions remain open, also we need some (gold)labels to test the functionality in the future.

**Comment by thuettel** [comment]
I just had a look at the goldlabel files from the fonduer tutorials. They do not seem to be to complicated, so it's probably worth a try to replicate a small gold label file based on ~ 5 documents and the dummy matchers for developing and testing of the pipeline. Open question is how big the sample size needs to be for the ML models to work properly.

**Comment by Wanille** [comment]
Dont have enough time unfortunately, just do it without gold labels still works :)

**Comment by macmacpri** [comment]
I have just worked through the supervision part of the tutorials again. Especially the wiki tutorial, this time with 30 docs, and I went through every step again. 

Copied both of our existing elaborations in the Branch 8-dummy-pipeline and added additional information/questions to the copies (marked with the abbreviation MP to distinguish them from our first elaboration). 

The copies are not intended for the dummy run!
